examples/all-clusters-app/esp32/main/DeviceWithDisplay.cpp

```diff
@@ -196,6 +196,36 @@ class EditAttributeListModel : public TouchesMatterStackModel
                 ESP_LOGI(TAG, "SystemMode changed to : %d", n);
                 app::Clusters::Thermostat::Attributes::OccupiedHeatingSetpoint::Set(1, n);
             }
+            else if (name == "Current Lift")
+            {
+                // update the current lift here for hardcoded endpoint 1
+                ESP_LOGI(TAG, "Current position lift percent 100ths changed to : %d", n);
+                app::Clusters::WindowCovering::Attributes::CurrentPositionLiftPercent100ths::Set(1, static_cast<uint16_t>(n * 100));
+            }
+            else if (name == "Current Tilt")
+            {
+                // update the current tilt here for hardcoded endpoint 1
+                ESP_LOGI(TAG, "Current position tilt percent 100ths changed to : %d", n);
+                app::Clusters::WindowCovering::Attributes::CurrentPositionTiltPercent100ths::Set(1, static_cast<uint16_t>(n * 100));
+            }
+            else if (name == "Opr Status")
+            {
+                // update the operational status here for hardcoded endpoint 1
+                ESP_LOGI(TAG, "Operational status changed to : %d", n);
+                app::Clusters::WindowCovering::Attributes::OperationalStatus::Set(1, static_cast<uint8_t>(n));
+            }
+            else if (name == "Bat remaining")
+            {
+                // update the battery percent remaining here for hardcoded endpoint 1
+                ESP_LOGI(TAG, "Battery percent remaining changed to : %d", n);
+                app::Clusters::PowerSource::Attributes::BatteryPercentRemaining::Set(1, static_cast<uint8_t>(n * 2));
+            }
+            else if (name == "Charge level")
+            {
+                // update the battery charge level here for hardcoded endpoint 1
+                ESP_LOGI(TAG, "Battery charge level changed to : %d", n);
```

Comment:
```suggestion
                ESP_LOGI(TAG, "Battery charge level changed to : %d%%", n);
```
might be clearer, so it's clear that we are logging the percentage, not the value we set for the attribute.

Reply:
Battery charge level is an enum for values OK, Warning and Critical. Changed this part so that it is more clear in the UI.